hack/uninstall.sh

```diff
@@ -12,6 +12,8 @@ oc delete namespaces/openshift-dns
 oc delete clusteroperator dns
 oc delete clusterroles/openshift-dns-operator
 oc delete clusterroles/openshift-dns
+oc delete clusterroles/dns-monitoring
```

Comment:
I see that clusterroles and clusterrolebindings are deleted but not roles and rolebindings. Is there a reason for this?

Reply:
Roles and RoleBindings are namespace scoped, so deleting 'openshift-dns-operator' namespace will delete these resources as well (https://github.com/openshift/cluster-dns-operator/blob/ed047bd2a2a6d74d45f5505275a5e7f5d1671874/hack/uninstall.sh#L10)